src/components/app/UrlManager.js

```diff
@@ -77,7 +77,9 @@ class UrlManager extends React.PureComponent<Props> {
       // Just silently finish the url setup and return to home.
       urlSetupDone();
     } else {
-      throw new Error('This if branch should not be reached.');
+      throw new Error(
+        'An an unhandled case was reached during the initial processing of URLs'
```

Comment:
Typo: you have "an" twice at the start of the sentence :)

Reply:
ahh, thanks! fixed it :)